SDC_DB/createSchema.js

```diff
@@ -30,21 +32,37 @@ const restaurantSchema = mongoose.Schema({
       tags: String,
     }],
   },
-}).index({ name: 'text' });
+}).index({ name: 1 });
 
 const Restaurant = mongoose.model('restaurantMenus', restaurantSchema);
 
 Restaurant.init()
   .then(() => mongoose.disconnect());
 
-function write10Million(start = 1e2) {
+const sampleDataGen = (i) => {
+  const data = {
+    id: i,
+    name: faker.company.companyName(),
+    menu: {
+      lunch: menu.entreeMenuGen(),
+      dinner: menu.entreeMenuGen(),
+      dessert: menu.dessertMenuGen(),
+    },
+  };
+  return data;
+};
+
+function write10Million(start = 1e7) {
   let i = start;
   let freeSpace = true;
 
   while (i > 0 && freeSpace) {
-    const data = generator(i);
+    const data = sampleDataGen(i);
     freeSpace = writeStream.write(`${JSON.stringify(data)}\n`);
```

Comment:
Is having a new line character necessary with the mongo import method? Just curious. I might have to implement that myself..